internal/util/projutil/project_util.go

```diff
@@ -253,6 +253,22 @@ func MustGetGopath() string {
 	return gopath
 }
 
+// PluginKeyToOperatorType converts a plugin key string to an operator project
+// type.
+// TODO(estroz): this can probably be made more robust by checking known
+// plugin keys directly.
+func PluginKeyToOperatorType(pluginKey string) OperatorType {
+	switch {
+	case strings.HasPrefix(pluginKey, "go"):
+		return OperatorTypeGo
+	case strings.HasPrefix(pluginKey, "helm"):
+		return OperatorTypeHelm
+	case strings.HasPrefix(pluginKey, "ansible"):
+		return OperatorTypeAnsible
+	}
+	return OperatorTypeUnknown
+}
```

Comment:
This function already exists in cmd/operator-sdk/generate/internal/genutil.go.

We should not duplicate it.

Reply:
@joelanford we are not duplicating. We moved from there to here since it is part of the project_utils domain and is useful outside of `cmd/operator-sdk/generate/`.

Reply:
I could totally be missing it, but I don't see the corresponding removal of this function from that package in this PR.

Reply:
You are not as always. I did this in another pr, pushed here the change. Really tks for the good catcher.